src/Contact/src/Delegator/ContactFormDelegator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Frontend\Contact\Delegator;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Exception\NotSupported;
+use Frontend\Contact\Entity\Department;
+use Frontend\Contact\Form\ContactForm;
+use Frontend\User\Entity\UserRole;
+use Laminas\ServiceManager\Factory\DelegatorFactoryInterface;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+class ContactFormDelegator implements DelegatorFactoryInterface
+{
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     * @throws NotSupported
+     */
+    public function __invoke(
+        ContainerInterface $container,
+        $name,
+        callable $callback,
+        ?array $options = null
+    ): ContactForm {
+
+        /** @var ContactForm $contactForm */
+        $contactForm = $callback();
```

Comment:
```suggestion
        $contactForm = $name();
```
No need to use the callback here, `$name` parameter containes the requested service name.